### PR TITLE
Add option to allow moderator to add member to closed circle without invitation step

### PR DESCRIPTION
**Pull request by tcoupin**
See feature request #288

### CODE REVIEW

File: lib/Model/Member.php [hunk 1]
```diff
@@ -112,6 +112,7 @@ private function inviteIntoClosedCircle() {
 			case Member::STATUS_NONMEMBER:
 			case Member::STATUS_KICKED:
 				$this->setStatus(Member::STATUS_INVITED);
+				//$this->setAsAMember(Member::LEVEL_MEMBER);
```
Comment (daita) [inline lib/Model/Member.php]: not needed

File: lib/Service/ConfigService.php [hunk 2]
```diff
@@ -90,6 +92,9 @@ class ConfigService {
 	/** @var int */
 	private $localNonSSL = -1;
 
+	/** @var int */
+	private $skipInvitationStep = -1;
```
Comment (daita) [inline lib/Service/ConfigService.php]: not needed

File: lib/Service/ConfigService.php [hunk 3]
```diff
@@ -157,6 +162,17 @@ public function isFederatedCirclesAllowed() {
 		return ($this->allowedFederatedCircles === 1);
 	}
 
+	/**
+	 * @return bool
+	 */
+	public function isInvitationSkipped() {
+		if ($this->skipInvitationStep === -1) {
+			$this->skipInvitationStep =
+				(int)$this->getAppValue(self::CIRCLES_SKIP_INVITATION_STEP);
```
Comment (daita) [inline lib/Service/ConfigService.php]: there is no need to store the value, getAppValue already cache it.

**Comment by daita** [comment]
Hello, haven't tested yet, but it looks good. Can you check my comments ?

**Comment by tcoupin** [comment]
@daita thanks for review. I've just update my PR

**Comment by tcoupin** [comment]
Hey @daita,
changelog for 0.17.6 contains "new settings to allow moderator to add member to closed circle without invitation step" but this PR is not merged ??

**Comment by daita** [comment]
yup, my bad

**Comment by daita** [comment]
i'll release 0.17.7 quickly !

**Comment by wiswedel** [comment]
resolves #288

**Pull request merged**